sklearn/multioutput.py

```diff
@@ -37,12 +39,90 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.copy(estimator)
+
+    if sample_weight is not None:
```

Comment:
Since the regressor and classifier is sharing same interface, so choice of argument for `classes` is non-neglect-able, but for `sample_weight` i might need to check, is it universal for sgd estimator